src/main/java/com/eprosima/fastdds/fastddsgen.java

```diff
@@ -640,7 +641,7 @@ public static void printHelp()
         System.out.println("\t\t" + cnames_arg + ": generates string and sequence types compatible with C.");
         System.out.println("\t\t" + ros2_names_arg + ": generates type naming compatible with ROS2.");
         System.out.println("\t\t" + version_arg + ": shows the current version of eProsima Fast DDS gen.");
-        System.out.println("\tand the supported input files are:");
+        System.out.println("\tThe supported input files are:");
```

Comment:
If change this, I will change "where the options are" with "The available options are".

Reply:
Changed in [8fc2a86](https://github.com/eProsima/Fast-DDS-Gen/pull/319/commits/8fc2a861bb7168060e6ace6f484914646c47d570)